issue_release_status.go

```diff
@@ -31,12 +54,85 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 
 	ctx, logger := githubapp.PreparePRContext(ctx, installationID, repository, prNum)
-
-	logger.Debug().Msgf("Event action is %s", event.GetAction())
 	if event.GetAction() != "created" {
 		return nil
 	}
 
+	prBase := event.GetPullRequest().GetBase().GetRef() // The branch which the pull request is ending.
+
+	// Retrieve auto-release-policy
+
+	serviceName := event.GetRepo().GetName()
+	servicePath := handler.releaseManagerURL + "/policies?service="
+
+	// Create client
+	client := &http.Client{}
+
+	// Set route
+	resp, err := client.Get(servicePath + serviceName)
+	if err != nil {
+		return errors.Wrap(err, "issueing a GET to specified URL")
+	}
+
+	req, err := http.NewRequest("GET", servicePath+serviceName, nil) // dublicate work??
+	if err != nil {
+		return errors.Wrap(err, "wrapping new request")
+	}
+
+	req.Header.Add("Authorization", "Bearer "+handler.releaseManagerAuthToken)
+
+	resp, err = client.Do(req)
+
+	var policyResponse ListPoliciesResponse
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	defer resp.Body.Close()
+
+	json.Unmarshal(body, &policyResponse)
+
+	var autoReleaseEnvironments []string
+	var botMessage string
+
+	// Not that easy to read, sry
+	if len(policyResponse.AutoReleases) > 0 {
```

Comment:
Go has an idiom about early returns. This helps with the concept of "line of sight" by keeping the happy path to the left. (Read more here: https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88)

So we would usually invert the `if` statement:
```suggestion
	if len(policyResponse.AutoReleases) == 0 {
		log.Debug().Msg("No auto-release policies was detected for the base branch")
		return nil
	}
	// do the rest
```

This also simplifies the code and thus the comment about readability can disappear 🙈